### PR TITLE
Refactor batch AIBrix runtime payload and Kubernetes execution flow

### PR DESCRIPTION
**Pull request by Jeffwan**
## Pull Request Description

  - Refactors extra_body.aibrix to use runtime.target and resource_allocation instead of the old compute /    planner_decision shape.

  - Updates Console planner submission to send the new AIBrix payload contract.

  - Removes storage/metastore from BatchProfile; profiles now focus on batch policy such as scheduling/quota.

  - Restores and improves Kubernetes runtime paths, including KubernetesJob support and Deployment runtime handling.

  - Updates batch smoke script to accept a full --aibrix-extra-body JSON payload.

  - Improves batch observability and error handling, including richer job-driver logs and mock app timing/log output.

  - Fixes cancellation timestamp handling and Redis env parsing edge cases.

  - Updates tests, samples, and docs for the new profile/runtime contract.

## Related Issues
Resolves: #[Insert issue number(s)]

**Important: Before submitting, please complete the description above and review the checklist below.**

---

<details>
<summary><strong>Contribution Guidelines (Expand for Details)</strong></summary>

<p>We appreciate your contribution to aibrix! To ensure a smooth review process and maintain high code quality, please adhere to the following guidelines:</p>

<h3>Pull Request Title Format</h3>
<p>Your PR title should start with one of these prefixes to indicate the nature of the change:</p>
<ul>
    <li><code>[Bug]</code>: Corrections to existing functionality</li>
    <li><code>[CI]</code>: Changes to build process or CI pipeline</li>
    <li><code>[Docs]</code>: Updates or additions to documentation</li>
    <li><code>[API]</code>: Modifications to aibrix's API or interface</li>
    <li><code>[CLI]</code>: Changes or additions to the Command Line Interface</li>
    <li><code>[Misc]</code>: For changes not covered above (use sparingly)</li>
</ul>
<p><em>Note: For changes spanning multiple categories, use multiple prefixes in order of importance.</em></p>

<h3>Submission Checklist</h3>
<ul>
    <li>[ ] PR title includes appropriate prefix(es)</li>
    <li>[ ] Changes are clearly explained in the PR description</li>
    <li>[ ] New and existing tests pass successfully</li>
    <li>[ ] Code adheres to project style and best practices</li>
    <li>[ ] Documentation updated to reflect changes (if applicable)</li>
    <li>[ ] Thorough testing completed, no regressions introduced</li>
</ul>

<p>By submitting this PR, you confirm that you've read these guidelines and your changes align with the project's contribution standards.</p>

</details>

### CODE REVIEW

File: python/aibrix/aibrix/batch/batch_manager.py [hunk 1]
```diff
@@ -775,17 +779,31 @@ async def mark_job_done(self, job_id: str) -> BatchJob:
...
                 )
             )
+        elif existing == ConditionType.CANCELLED:
+            job.status.cancelled_at = now
+        elif existing == ConditionType.FAILED:
+            job.status.failed_at = now
+        elif existing == ConditionType.EXPIRED:
+            job.status.expired_at = now
+        else:  # COMPLETED already recorded
+            job.status.completed_at = now
```
[the remark this answers is not in the document]
Comment (Jeffwan) [reply python/aibrix/aibrix/batch/batch_manager.py:806]: It always return "Optional[ConditionType]", no need to worry about it

File: python/aibrix/aibrix/batch/manifest/storage_env.py [hunk 2]
```diff
@@ -12,261 +12,126 @@
...
             {
-                "name": "STORAGE_AWS_ENDPOINT_URL",
-                "value": profile.spec.storage.endpoint_url,
+                "name": "STORAGE_LOCAL_PATH",
+                "value": os.getenv("WORKER_STORAGE_LOCAL_PATH")
+                or os.getenv("STORAGE_LOCAL_PATH")
+                or ".storage",
             }
         )
     return env
```
[the remark this answers is not in the document]
Comment (Jeffwan) [reply python/aibrix/aibrix/batch/manifest/storage_env.py:109]: fixed

File: development/app/app.py [hunk 3]
```diff
@@ -159,6 +171,34 @@ def create_vllm_error(message, error_type, status_code):
...
+def _sleep_for_request_latency(started_at, simulated_latency):
+    overhead = datetime.now().timestamp() - started_at
+    if simulated_latency > overhead:
+        time.sleep(simulated_latency - overhead)
+    elif simulated_latency > 0.0:
+        logger.warning(
+            f"Latency is less than overhead: L{simulated_latency} - O{overhead}"
+        )
+    elif MOCK_REQUEST_DURATION_SECONDS > overhead:
+        time.sleep(MOCK_REQUEST_DURATION_SECONDS - overhead)
```
[the remark this answers is not in the document]
Comment (Jeffwan) [reply development/app/app.py:199]: fixed
